src/print-1-10-growing.py

```diff
@@ -1,2 +1,7 @@
 
 # Print the numbers described in the exercise
+j=[]
+for i in range(1,11):
+    j.append(i)
+    print(j)
```

Comment:
This prints the numbers, but not quite in the format, we want them.

Consider a list with the numbers 1, 2, and 3. Printing it will look like this:

```python
>>> x = [1, 2, 3]
>>> print(x)
[1, 2, 3]
```

We don't want the brackets, nor the commas. We just want

```
1 2 3
```